tests/fs-test.rs

```diff
@@ -24,6 +25,21 @@ pub fn verify_file_equals(testfile: &PathBuf, contents: &str) {
 	assert_eq!(file_content, contents.to_string());
 }
 
+/// Wrapper that builds test kernels and runs the VM in a new thread.
+/// Useful for testing Landlock.
+///
+/// * `bin_path` - Path of kernel to be run.
+/// * `params` - Params to run the VM with.
+#[allow(dead_code)]
+pub fn run_vm_in_thread(bin_path: PathBuf, params: Params) -> VmResult {
+	thread::spawn(move || {
+		let vm = UhyveVm::new(bin_path, params).unwrap();
+		vm.run(None)
+	})
+	.join()
+	.expect("Uhyve thread panicked.")
+}
+
```

Comment:
Can we move this to `common.rs`?

Reply:
sure

Reply:
hang on - `clippy` is being annoying because we don't use `UhyveVm` in many cases under `fs-test` now.